packages/repo-mock/src/index.ts

```diff
@@ -22,15 +37,23 @@ export function mockRepo(baseURL: string, targets: Target[]): string {
   const timestampMeta = createTimestampMeta(snapshotMeta, keyPair);
   const rootMeta = createRootMeta(keyPair);
 
+  // Calculate paths for all of the metadata files
+  const rootPath = `${metadataPrefix}/1.root.json`;
```

Comment:
maybe we should path.join?

Reply:
These are for the remote URLs, so `path.join` wouldn't actually be appropriate.